CMakeLists.txt

```diff
@@ -81,6 +82,7 @@ add_subdirectory(external)
 
 # Build the main chimera executable.
 add_executable("${PROJECT_NAME}"
+  ${chimera_headers}
```

Comment:
Why do we need to do this here?  Is there a platform that requires the headers as sources?

Reply:
Not required but useful when using Qt Creator. Open to remove this.